app/mailers/emailer.rb

```diff
@@ -18,21 +29,13 @@ def submission_confirmation(params)
 
   def confirmation_instructions(record, token, opts = {})
     contact = AccountService.get(record.salesforce_contact_id)
-    @name = if contact.present?
-              "#{contact['firstName']} #{contact['lastName']}"
-            else
-              record.email
-            end
+    @name = name(contact, record)
```

Comment:
aha, we both refactored this function in each of our PRs 😄 

we can merge it afterwards when they're combined.